### DISCUSSION TITLE
Results of the single technosphere elements of an activity

### DISCUSSION THREAD

**Discussion by ezio95** [LCA-ActivityBrowser/activity-browser#1005] [Q&A] [answered]
Hello everybody!

I have created an activity on AB and would like to display the impact of every element of the technosphere, how can I do this?

![grafik](https://github.com/LCA-ActivityBrowser/activity-browser/assets/140547889/d83e1e63-5a64-4039-a1cc-37f5c1dfb4ee)

![grafik](https://github.com/LCA-ActivityBrowser/activity-browser/assets/140547889/67a09091-dd6c-4ea0-8b06-c43809c1e746)

I've tried clicking "EF and Process Contribution" but none of the "Aggregate by"-Categories gets the desired elements.,

![grafik](https://github.com/LCA-ActivityBrowser/activity-browser/assets/140547889/8f34474c-6fc6-40a8-bd7c-dd907e810308)

Is there a way to display them? or is the only alternative to manually load each element of the technosphere into the LCA setup?

Thank you in advance!

**Comment by marc-vdm** [comment]
Hey thanks for your question! 

Could you explain what you mean exactly with 'element'? Are you looking for atomic elements (`Fe`, `Ni`, `Li`, `Co` etc)? Or do you mean each product going into your battery (`battery cell, Li-ion, NMC111`, `copper, anode`, `controller, for electric scooter` etc)? Or do you mean something else entirely?

**Reply by andreadm84** [reply]
Hello Marc!
I actually have the same question above (if I understood it correctly...). 

Would it be possible to aggregate the results to display **only** the contribution of the products going into the battery (battery cell, Li-ion, NMC111, copper, anode, controller, for electric scooter, etc, not the background processes), as in the graph in the example below?  

Many thanks in advance!

Andrea


![example](https://github.com/LCA-ActivityBrowser/activity-browser/assets/114665236/e1fa9426-5748-4f70-96fc-e326d6648706)

**Reply by marc-vdm** [reply]
Hi Andrea,

Unfortunately that's currently not possible to see in AB, though it is something I'm working on adding, but this is a _long term_ project, so don't expect anything on this soon. One alternative you could use currently is the Sankey diagram in Activity Browser, I think that should show something similar to what you want to see.

However, just for my understanding: there are 2 ways to see what you're interested in:
1: You can show only the direct emissions from those processes you mention (e.g. the CO2 emitted directly in the `copper, anode` process), this would also have a 'remainder' group to account for 100% of the impact.
2: The life cycle stage impact which processes are responsible for (e.g. the CO2 emitted by all processes that contribute to the manufacturing of the copper anode (e.g. the CO2 from electricity production needed for only the copper production etc). This would show the full impact of that you're looking at, grouped into life cycle stages.

Which of the two would you find most interesting?

**Reply by andreadm84** [reply]
Hello Marc, thanks a lot for your reply, and, especially, for working on solving this issue!

Indeed, I see that in Sankey I can find the pieces of information I am looking for, although they cannot be imported as a table (therefore I would need to copy them manually for each impact category). 

About your question, are you asking if it would be better to show only emissions due to foreground activities and show the background activities as "rest" (way 1), or to show the full life cycle, including the background activities (way 2)?  
If I understand it well, the best (in my opinion, ofc) is to show the total emissions from each of the inputs directly connected to my process, as shown in the figure in my previous message. This provides an indication of where to look further in my study (if for instance "copper, anode production" has the highest emissions in my battery production activity, I can perform another LCA only on "copper, anode production", to see the source of such emission). This is how the results are presented in most of the LCA software (Simapro, Gabi, OpenLCA).  
Therefore, i guess i am more towards option 2 (hope i understand well your question).

Let me know if you would need any further information, or do you have any alternative solution to propose :)
Thanks again,
Andrea

**Reply by marc-vdm** [reply]
Thanks a lot for the info!

**Comment by marc-vdm** [comment]
Hi @ezio95 and @andreadm84,

It's a work in progress, but we have an implementation here in #1046. It will likely come to `2.10`, but it will be a while before that is ready.

**Comment by marc-vdm** [comment, marked as answer]
@ezio95 @andreadm84 
This is now possible with [First Tier contributions](https://github.com/LCA-ActivityBrowser/activity-browser/wiki/LCA-Results#first-tier-ft-contributions), which is available since Activity Browser [`2.11.0`](https://github.com/LCA-ActivityBrowser/activity-browser/releases/tag/2.11.0).
